README.md

```diff
@@ -62,57 +31,66 @@ gist.create({
    }
 }).then(function({data}) {
    // Promises!
-   let gistJson = data;
-   gist.read(function(err, gist, xhr) {
-      // if no error occurred then err == null
-
-      // gistJson === httpResponse.data
-
-      // xhr === httpResponse
-   });
+   let createdGist = data;
+   return gist.read();
+}).then(function({data}) {
+   let retrievedGist = data;
+   // do interesting things
 });
 ```
 
 ```javascript
-import GitHub from 'github-api';
+var GitHub = require('github-api');
```

Comment:
sure, but `var` is not deprecated in anyway, so if we can help people understand the repo, I think it's best, regardless of wether it's matching style guide or not, don't you think? (see #392)